src/main/java/calculator/controller/CalculatorController.java

```diff
@@ -0,0 +1,24 @@
+package calculator.controller;
+
+import calculator.domain.InputStringSeparator;
+import calculator.domain.SumCalculator;
+import calculator.view.InputView;
+import calculator.view.OutputView;
+
+public class CalculatorController {
+    private final InputView inputView;
+    private final OutputView outputView;
+    private final SumCalculator sumCalculator;
+
+    public CalculatorController() {
+        inputView = new InputView();
+        outputView = new OutputView();
+        sumCalculator = new SumCalculator();
+    }
+
+    public void startCalculator() {
+        outputView.printStartMessage();
+        InputStringSeparator inputStringSeparator = new InputStringSeparator(inputView.getString());
+        outputView.printResult(sumCalculator.getResult(inputStringSeparator.getInputNumber()));
```

Comment:
로직 3개가 겹쳐져 있어요. 변수로 분리해주면 더 읽기 쉬울 것 같아요

```suggestion
    var seperatedNumber = inputStringSeparator.getInputNumber();
    var result = sumCalculator.getResult(seperatedNumber);
    outputView.printResult(result);
```

Reply:
가독성이 더 좋아졌습니다 감사합니다 :)